project/GenerateAnyVals.scala

```diff
@@ -232,6 +234,7 @@ import scala.language.implicitConversions"""
       "@unboxImpl@" -> "???"
     )
     def interpolations = Map(
+      "@article@"   -> (if (this == I) "an" else "a"),
```

Comment:
How clever. Conflicts with https://github.com/scala/scala/pull/6490/files where I seem to have hit a couple of more spots.